### DISCUSSION TITLE
Not able to See blocks data on blockscout.

### DISCUSSION THREAD

**Discussion by gautamjha2002** [blockscout/blockscout#9587] [Support]
I deployed besu provate network, and for this I want to enable blockscout as explorer.

I deployed everything with docker compose

I changed  ENV variable in docker-compose.yaml file

and run docker compose up --build -d 

everything got deployed, 9 containers are up and running, on port 80 I was able to get the UI of blockscout. But there is not any information about blocks, transactions etc nothing 

When I try to debug it i came accross logs of db and see this (Screenshot pasted below)

![Screenshot from 2024-03-08 19-10-36](https://github.com/blockscout/blockscout/assets/98412293/563d5c93-5908-485c-bc9f-ca508aa7de31)

Please help me out here how can i debug it, what are the possible solution for this??

I checked some discussion (issues) and I got a similar one but didn't found the solution 

Issue I found :- https://github.com/blockscout/blockscout/issues/9170

Please help me out here.

**Comment by GimluCom** [comment]
Make sure you have set `NEXT_PUBLIC_APP_HOST` and `NEXT_PUBLIC_API_HOST` in [envs/common-frontend.env](https://github.com/blockscout/blockscout/blob/master/docker-compose/envs/common-frontend.env).
All ENVs: https://docs.blockscout.com/for-developers/information-and-settings/env-variables
